transcoder/main.go

```diff
@@ -217,11 +218,12 @@ func (h *Handler) GetAttachment(c echo.Context) error {
 		return err
 	}
 
-	ret, err := h.metadata.GetAttachmentPath(sha, false, name)
+	ret, err := h.metadata.GetAttachment(sha, name)
+	defer ret.Close()
 	if err != nil {
 		return err
 	}
-	return c.File(ret)
+	return c.Stream(200, "TODO", ret)
```

Comment:
TODO that should be fixed before merging

Reply:
any idea what the proper MIME types for these + attachments should be? I think they can both vary depending on what extension the underlying file uses.

Reply:
this should be fixed in the latest commit